### PR TITLE
Blood vomiting reduces bloodlevels & retarded pneumatic cannons. (meant to be a two seperate pulls, mistakes were made.)

### PR DESCRIPTION
**Pull request by B1gb3ast**
Fluff stuff. And a minor buff to viro.

Not tested. should be tested, probably. But I couldn't test. so yeah. Someone test it.
Git is so annoying sometimes.

### CODE REVIEW

File: code/datums/diseases/advance/symptoms/vomit.dm [hunk 1]
```diff
@@ -92,6 +92,9 @@ Bonus
 	var/brute_dam = M.getBruteLoss()
 	if(brute_dam < 50)
 		M.adjustBruteLoss(3)
+		if(istype(M, /mob/living/carbon/human))
+			var/mob/living/carbon/human/H
```
Comment (oisin100) [inline code/datums/diseases/advance/symptoms/vomit.dm]: You dont initialize the variable!

Initialize it with "var/mob/living/carbon/human/H = M"

**Comment by B1gb3ast** [comment]
upped the drip a little.

**Comment by B1gb3ast** [comment]
oops. meant to make another pull.

**Comment by B1gb3ast** [comment]
Screw it they both go here

**Comment by oisin100** [comment]
You made this PR for the master branch... You need to create pulls to the dev branch so that it can be tested before its added to the server. You can do this by closing the Pr and reopening it with the right branch selected

File: code/game/objects/items/weapons/pneumaticCannon.dm [hunk 2]
```diff
@@ -142,3 +160,54 @@
...
+
+
+//Because adam said so
+/obj/item/weapon/pneumatic_cannon/bluespace
+	name = "bluespace cannon"
+	desc = "An enigmatic, infinitely deep cannon that can fire any object loaded into it. Pulling the trigger will discharge the bluespace compartmentalizer, handle with care."
+	w_class = 5
+	maxWeightClass = 999999
+	gasPerThrow = 0
+	obj/item/weapon/tank/internals/tank = 1
```
Comment (X-TheDark) [inline code/game/objects/items/weapons/pneumaticCannon.dm:172]: What's this supposed to mean? Shouldn't this not only be a var, but actually instantiated as well?

Comment (B1gb3ast) [inline code/game/objects/items/weapons/pneumaticCannon.dm:172]: Probably but I have no idea what im doing

**Comment by B1gb3ast** [comment]
Leave a comment

**Pull request closed without merging**